### ISSUE TITLE
parsetab.py still used?

### ISSUE THREAD

**Issue by ccoffrin** [QISKit/qiskit-sdk-py#69]
A few parts of the code mention `parsetab.py`, which seems to be auto generated.

I ran into an issue where I installed `qiskit` using `pip3` on an admin account and then switched to a non-admin account for using the library.  When running `QuantumProgram.execute` I get the following warning,

```
WARNING: Couldn't create 'parsetab'. [Errno 13] Permission denied: '/usr/local/lib/python3.6/site-packages/qiskit/qasm/parsetab.py'
WARNING: Couldn't create 'parsetab'. [Errno 13] Permission denied: '/usr/local/lib/python3.6/site-packages/qiskit/qasm/parsetab.py'
```

**Comment by atilag** [comment]
Thanks for the report Carleton. Even though we encourage the use of `virtualenvs` to all our users, there may be cases where this is not an option. Fortunately, I think there's a way to fix this without any impact :)
I'll work on it.

**Comment by atilag** [comment]
Bug has been fixed in the new available pip package version: 0.3.6
Thanks for the report @ccoffrin !

**Comment by ccoffrin** [comment]
Thanks a lot!
